src/SourceBuild/content/repo-projects/installer.proj

```diff
@@ -81,4 +84,63 @@
     <EnvironmentVariables Include="CLIBUILD_SKIP_BUNDLEDDOTNETTOOLS=true" />
   </ItemGroup>
 
+  <Target Name="GetInputsOutputForCreatePrivateSourceBuiltArtifactsArchive"
+          DependsOnTargets="DetermineSourceBuiltSdkVersion">
+    <!-- Inputs: Packages to include in the tarball -->
+    <ItemGroup>
+      <ArtifactsPackageToBundle Include="$(ArtifactsShippingPackagesDir)**;
+                                         $(ArtifactsNonShippingPackagesDir)**"
+                                Condition="!$([System.String]::Copy('%(Identity)').EndsWith('.symbols.nupkg'))" />
+      <ReferencePackageToBundle Include="$(ReferencePackagesDir)**"
+                                Condition="!$([System.String]::Copy('%(Identity)').EndsWith('.symbols.nupkg'))" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <!-- Create a layout directory for the files that are to be included in the artifacts tarball. -->
+      <SourceBuiltLayoutDir>$([MSBuild]::NormalizeDirectory('$(BaseIntermediateOutputPath)', 'artifacts-layout'))</SourceBuiltLayoutDir>
```

Comment:
I know this was existing logic, but should this directory be proactively cleaned up?

Reply:
Not necessarily. All the files in that directory are symbolic links except for the .versions and the PackageVersions.props file. That directory isn't much different than the semaphore files that we already store in the `BaseIntermediateOutputPath`. The `Clean` target is the actor that is responsible for cleaning the intermediate artifacts.

Reply:
Right. I purposefully wanted it to not be cleaned up so devs can evaluate its contents if something goes wrong. It essentially has no impact on disk size since it's just symlinks.